### PR TITLE
Added information to set affinity documentation about unsupported subgraphs 

### PR DESCRIPTION
**Pull request by ilyachur**

### CODE REVIEW

**Review by eaidova — APPROVED** [review]

File: docs/IE_DG/supported_plugins/HETERO.md [hunk 1]
```diff
@@ -30,6 +30,8 @@ Another way to annotate a network is to set affinity manually using <code>ngraph
 
 @snippet openvino/docs/snippets/HETERO0.cpp part0
 
+> **NOTE**: If you set affinity manually, be careful at the current moment Inference Engine plugins don't support constant (`Constant`->`Result`) and empty (`Parameter`->`Result`) networks. Please avoid such subgraphs when you set affinity manually.
+
 The fallback policy does not work if even one layer has an initialized affinity. The sequence should be calling of automating affinity settings and then fix manually.
```
Comment (eaidova) [inline docs/IE_DG/supported_plugins/HETERO.md:35]: I think logically this note should be after paragraph  about all layers should have affinity

Comment (ilyachur) [reply docs/IE_DG/supported_plugins/HETERO.md:33]: Fixed

**Comment by ilyachur** [comment]
@apankratovantonp Please have a look. What logic do we have inside the hetero plugin? Can we get the constant network after graph splitting?

**Comment by apankratovantonp** [comment]
> @apankratovantonp Please have a look. What logic do we have inside the hetero plugin? Can we get the constant network after graph splitting?

Yes HETERO plugin can produce unsupported subnetworks. 
Today all plugins solve this problem inside the QueryNetwork implementations.

**Review by apankratovantonp — APPROVED** [review]

**Comment by ilyachur** [comment]
> Yes HETERO plugin can produce unsupported subnetworks.
> Today all plugins solve this problem inside the QueryNetwork implementations.

Am I right that at the current moment plugin is responsible for avoiding such issues?

**Comment by apankratovantonp** [comment]
> > Yes HETERO plugin can produce unsupported subnetworks.
> > Today all plugins solve this problem inside the QueryNetwork implementations.
> 
> Am I right that at the current moment plugin is responsible for avoiding such issues?

Formally there is no plugin responsibilities to avoid unsupported cases. But in fact yes it is.

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**